src/lib/webpack-config.js

```diff
@@ -105,8 +105,8 @@ export default env => {
 					pkg = readJson(manifest) || {};
 				return !!(pkg.module || pkg['jsnext:main']);
 			},
-			babelrc: false,
-			...createBabelConfig(env)
+			babelrc: true,
```

Comment:
We can't have this. This is the reason for #9. Even though it's been set to `false`, it wasn't actually making it to the babel compiler.

I have a PR incoming for this.